traffic-participants.rst

```diff
@@ -0,0 +1,96 @@
+The traffic participants are co-simulated.
+
+The following figure shows a very simple use case.
+
+.. image:: _static/images/osi-traffic-participant-use-case-1.png
+   :align: center
+   :alt: Simple traffic participant
+
+The traffic participant bases its behavior only on an idealized view of the area around it.
+If they exist, the traffic participant’s dynamics are included in the model.
+The map, the simulated world state, the sensor view, and the traffic updates represent the ground truth, meaning objective statements about the world.
+The scenario and the traffic commands on the other hand represent the intentions of the traffic participants.
+
+The following figure shows a traffic participant with separately modeled behavior and dynamics.
+
+.. image:: _static/images/osi-traffic-participant-use-case-2.png
+   :align: center
+   :alt: Traffic participants with separate dynamics
+
+Separating dynamics from behavior introduces two more types of channels:
+
+* Measured data, meaning ground truth as interpreted by the traffic participant
+* Actuator-intention data, providing information on what actuators are supposed to do
+
+OSI currently provides only limited support for data structures describing measured internal states of the traffic participant.
+Actuator intentions are currently not covered by OSI and must be handled with a different data description format.
+
+The following figure shows a more complex traffic participant.
+
+.. image:: _static/images/osi-traffic-participant-use-case-3.png
+   :align: center
+   :alt: Traffic participant with sensor models, AD function, and dynamic model
+
+This use case will probably be relevant for modeling the ego vehicle, which includes the system-under-test.
+The traffic participant includes an arbitrary number of sensor models, which provide sensor data.
+Sensor data is another example of measured data as an interpretation of ground truth by the traffic participant.
+In contrast to data describing the internal state of the vehicle, sensor data describes how sensors interpret the surroundings, for example, with lidar, camera, radar, or ultrasonic technology.
+The AD function consumes measured data and outputs actuator-intention data.
+OSI currently does not support actuator-intention data.
+
+The following figure shows a cooperative use case with both an AD function and a human driver.
+
+.. image:: _static/images/osi-traffic-participant-use-case-4.png
+   :align: center
+   :alt: Traffic participant with an AD function and a human driver
+
+It is possible to model a traffic participant with an AD function in the loop, but a human driver can still override the actuation command.
+Such a cooperative use case is, for example, relevant to studies on human-machine interaction.
+OSI's limitations regarding dynamic-model input apply in this example as well.
+
+The following figure shows a similar example with the difference that there is only a human driver in control.
```

Comment:
Maybe also mention what a mock-up is: that you look on a screen where the virtual scenario is happening...

Reply:
- [ ] Explain "mock-up"

Reply:
sounds good.